source/_integrations/yale.markdown

```diff
@@ -27,13 +27,15 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The **Yale** integration allows you to integrate your [Yale](https://yale.com/) and some Yale Access devices in Home Assistant.
+The **Yale** integration allows you to integrate your [Yale](https://www.assaabloy.com/vn/en/solutions/products/yale) and some Yale Access devices in Home Assistant.
+
+The login credentials used to authenticate the integration are the same as the ones used in the Yale Home app.
```

Comment:
But how do you get the client id and secret for application credentials?

Reply:
Its account link so they don't need this

Reply:
Is there no way to set up an individual application?

If it's only account link it means this integration requires the cloud integration to be setup (subscription isn't required).

Reply:
We don't have a partnership with Fortune Brands Innovations (FBIN) aka August / Yale NA brand so that still uses the legacy auth / august integration.  Hopefully we can migrate that to oauth in the future but thats less of a technical issue and more of an operational negotiation to get a partner account with them.

Reply:
> Is there no way to set up an individual application?
> 
> If it's only account link it means this integration requires the cloud integration to be setup (subscription isn't required).

Right.  Do we need to add `cloud` to `dependencies` or `after_dependencies` ?

Reply:
https://github.com/home-assistant/core/pull/124821

Reply:
Since it's an always required dependency it needs to go into `dependencies`.

Reply:
Looks like we need to do some more mocking in the tests since there aren't any mocks for the cloud case.  I haven't found another integration that uses them so I may have to build them from scratch